fractal_server/main.py

```diff
@@ -88,6 +88,7 @@ async def _create_first_user(
     password: str,
     is_superuser: bool = False,
     slurm_user: Optional[str] = None,
+    slurm_accounts: Optional[list[str]] = None,
```

Comment:
Check that this function is called only once (and that is in this file), and remove all unused arguments: `slurm_user`, `slurm_accounts`, `cache_dir`.